src/pages/layout/Search/Index.tsx

```diff
@@ -101,6 +107,45 @@ export default function HeaderSearch() {
     }
   };
 
+  const handleSubmitSearch = async () => {
+    const accountPromise = getAccount(
+      {address: inputValue},
+      state.network_value,
+    )
+      .then(() => {
```

Comment:
You could use async / await as before, the request will start when you call the function.

Reply:
Sorry I'm not a JS expert. I'm a bit confused. I thought we wanted to do 2 queries simultaneously. Would you give me more instruction or code snippet?